xdebug.c

```diff
@@ -569,7 +569,10 @@ PHP_MINIT_FUNCTION(xdebug)
 		return SUCCESS;
 	}
 
-	xdebug_library_minit();
+	if (XDEBUG_MODE_IS(XDEBUG_MODE_TRACING) || XDEBUG_MODE_IS(XDEBUG_MODE_COVERAGE)) {
```

Comment:
This xdebug_library_minit function overrides a number of opcode handlers but the vast majority of them are only used during tracing or code coverage. The only one that is used during debugging is the INCLUDE_OR_EVAL code. So we only call this function if we are tracing or doing coverage. The INCLUDE_OR_EVAL opcode will be overridden separately later for debugging